src/cmd/viper.go

```diff
@@ -213,7 +213,7 @@ func setDefaults() {
 	v.SetDefault(VLogFormat, string(logger.FormatConsole))
 
 	// Package defaults that are non-zero values
-	v.SetDefault(VPkgOCIConcurrency, 3)
+	v.SetDefault(VPkgOCIConcurrency, 6)
```

Comment:
Why this change? Is ORAS more performant?

Reply:
Crane pushes and pulls all layers concurrently always, and had no way to respect this flag so for container operations the concurrency is actually decreasing. 

OCI concurrency flag is used for Zarf package OCI operations as well. The ORAS CLI defaults to three, so that's what we went with originally for Zarf packages, but I think three is conservative. Skopeo for example, goes with six and that feels like a more sane inbetween.

Reply:
won't block, but prob a good idea to call out in release notes the default has increased

there's a higher likelihood with 6 coroutines of pulling multiple large layers at a time, saturating a users network connection + causing higher memory usage